pkg/validate/raw/validate/management_annotation.go

```diff
@@ -15,17 +15,19 @@
 package validate
 
 import (
+	"kpt.dev/configsync/pkg/core"
 	"kpt.dev/configsync/pkg/importer/analyzer/ast"
 	"kpt.dev/configsync/pkg/importer/analyzer/validation/nonhierarchical"
 	"kpt.dev/configsync/pkg/metadata"
 	"kpt.dev/configsync/pkg/status"
 )
 
 // ManagementAnnotation returns an Error if the user-specified management annotation is invalid.
+// The only valid value in the source is `disabled`.
 func ManagementAnnotation(obj ast.FileObject) status.Error {
-	value, found := obj.GetAnnotations()[metadata.ResourceManagementKey]
-	if found && (value != metadata.ResourceManagementDisabled) {
-		return nonhierarchical.IllegalManagementAnnotationError(obj, value)
+	if !metadata.IsManagementDisabled(obj) && !metadata.IsManagementUnspecified(obj) {
```

Comment:
Could this be simplified to `metadata.IsManagementEnabled`? Is the concern that we may add more enum values?

Not trying to be difficult but I find this a bit harder to read than before. Maybe a switch would make it easier to read?

Reply:
IsManagementEnabled isn't the same thing. Disabled and Unspecified are allowed. Enabled is not allowed in the source and neither are any other values.

Reply:
Right, my suggestion was to replace:
`!metadata.IsManagementDisabled(obj) && !metadata.IsManagementUnspecified(obj)`
with 
`metadata.IsManagementEnabled(obj)`

But ack if we need to specifically handle the case of an invalid (non-enumerated) value

I think either using a switch or applying De Morgan's would help. It might just be my brain but I have a harder time reasoning about negated conjunctions.

e.g. maybe something like:
```golang
func ManagementAnnotation(obj ast.FileObject) status.Error {
  if metadata.IsManagementDisabled(obj) || metadata.IsManagementUnspecified(obj) {
    return nil
  }
  return nonhierarchical.IllegalManagementAnnotationError(obj,
    core.GetAnnotation(obj, metadata.ManagementModeAnnotationKey))
}
```

Reply:
I can follow up with a separate change to refactor this function.

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1636